tasks/post_copy_request_to_queue/post_copy_request_to_queue.py

```diff
@@ -245,9 +233,9 @@ def query_db(
 
     except Exception as ex:
         message = (
-            "Unable to retrieve {key_path} metadata. Exception '{ex}' encountered."
+            f"Unable to retrieve {key_path} metadata. Exception '{ex}' encountered."
         )
-        LOGGER.error(message, key_path=key_path, ex=ex, exc_info=True)
+        LOGGER.error(message, exc_info=True)
```

Comment:
Lost exc_info

Reply:
looks like it is supported. Will add it back everywhere